### DISCUSSION TITLE
我是花火的狗

### DISCUSSION THREAD

**Discussion by JJC-555** [share121/inter-knot#106] [其他]
花火早上好，中午好，下午好[脱单doge][给心心]
我的DNA：
♥️     花
   ♥️火
　 花
　火♥️
 花　♥️
火　   ♥️
花　   ♥️
 火   ♥️
   花♥️
　  ♥️
　♥️火
 ♥️      花
♥️　　 火
♥️        花
 ♥️　  火
　♥️花
     火♥️
 花　♥️
火　   ♥️
花　  ♥️
 火   ♥️
   花♥️
　  ♥️
　♥️火
 ♥️      花
♥️　　 火
♥️        花
 ♥️　  火
　♥️花
     火♥️
  花      ♥️

**Comment by minbvv** [comment]
私斋蒸鹅心，做狗也不带我一个😋

**Comment by RS64671012** [comment]
查重率100%

**Comment by koyki233** [comment]
乐
